service/controller/v17/resource/ipam/create.go

```diff
@@ -0,0 +1,167 @@
+package ipam
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"reflect"
+
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/ipam"
+	"github.com/giantswarm/microerror"
+	apismetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/aws-operator/service/controller/v17/controllercontext"
+	"github.com/giantswarm/aws-operator/service/controller/v17/key"
+)
+
+// EnsureCreated allocates guest cluster network segment. It gathers existing
+// subnets from existing AWSConfig/Status objects and existing VPCs from AWS.
+func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	customObject, err := key.ToCustomObject(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	r.logger.LogCtx(ctx, "level", "debug", "message", "checking if subnet needs to be allocated for cluster")
```

Comment:
It would be nice to have logs more structured. Like `finding out if subnet needs to be allocated for the cluster`, `found out ...`, `did not find out ...`. See https://github.com/giantswarm/fmt/blob/master/go/log_messages.md

Reply:
Refined wording. Better now or do you think other log entries should also be changed?

Reply:
That's cool. As I saw it in other cases are present continuous and past tense pairs. That makes it easier for me to match logs. Thanks!